components/template/Faq.vue

```diff
@@ -0,0 +1,66 @@
+<template>
+  <section class="container py-2 py-lg-5">
+    <h2 class="text-center section__heading">Frequently Asked Questions</h2>
+    <div class="accordion accordion-flush my-5" id="accordionFaqs">
+      <div v-for="(faq, index) in faqs" :key="index" class="accordion-item">
+        <h2 class="accordion-header" :id="`heading${index}`">
```

Comment:
Why :id="`heading${index}`"?

Reply:
we are using bootstrap accordion to differentiate which accordion to open. So index will differentiate the accordion

Reply:
But I see below that on the button you are already providing data-bs-target, so is the above id required? Can you check once if it works even after removing it?

Reply:
Yes its working